deps/exokit-bindings/webaudiocontext/src/AudioContext.cpp

```diff
@@ -9,14 +9,16 @@ lab::AudioContext *getDefaultAudioContext(float sampleRate) {
     _defaultAudioContext = lab::MakeRealtimeAudioContext(sampleRate);
 
     atexit([]() {
-      _defaultAudioContext.reset();
+      // NOTE: There was no reset method for AudioContext!
+      //_defaultAudioContext.reset();
```

Comment:
My thinking is we should reset the pointer to call the `AudioContext` destructor as intended. I think the issue is simply using the wrong signature, or wrong C++ version in compiler.